search-app/docs/api.md

```diff
@@ -1686,33 +1686,46 @@ Find collections matching service concept id.
 ##### <a name="c-polygon"></a> Polygon
 
 Polygon points are provided in counter-clockwise order. The last point should match the first point to close the polygon. The values are listed comma separated in longitude latitude order, i.e. lon1, lat1, lon2, lat2, lon3, lat3, and so on.
+This parameter supports the and/or option as shown below.
 
     curl "%CMR-ENDPOINT%/collections?polygon=10,10,30,10,30,20,10,20,10,10"
 
+    curl "%CMR-ENDPOINT%/collections?polygon=10,10,30,10,30,20,10,20,10,10&polygon=11,11,31,11,31,21,11,21,11,11&options[polygon][or]=true"
```

Comment:
In all of these examples I think you need `[]` after the spatial param. 

curl "%CMR-ENDPOINT%/collections?polygon[]=10,10,30,10,30,20,10,20,10,10&polygon[]=11,11,31,11,31,21,11,21,11,11&options[polygon][or]=true"